examples/jupyter_lite_config.json

```diff
@@ -7,6 +7,8 @@
       "https://github.com/conda-forge/releases/releases/download/noarch/ipycytoscape-1.2.2-pyhd8ed1ab_0.tar.bz2/ipycytoscape-1.2.2-pyhd8ed1ab_0.tar.bz2",
       "https://github.com/conda-forge/releases/releases/download/noarch/ipyleaflet-0.14.0-pyhd8ed1ab_1.tar.bz2/ipyleaflet-0.14.0-pyhd8ed1ab_1.tar.bz2",
       "https://github.com/conda-forge/releases/releases/download/noarch/ipympl-0.7.0-pyhd8ed1ab_0.tar.bz2/ipympl-0.7.0-pyhd8ed1ab_0.tar.bz2",
+      "https://github.com/conda-forge/releases/releases/download/noarch/ipyvue-1.5.0-pyhd3deb0d_0.tar.bz2/ipyvue-1.5.0-pyhd3deb0d_0.tar.bz2",
```

Comment:
One downside of the tarball URL approach might be that we can miss dependencies such as `ipyvuetify` depending `ipyvue`?

As defined here: https://github.com/mariobuikhuizen/ipyvuetify/blob/898a2668e1773701881252e6f96e30ef3cb436f0/setup.py#L158-L160